src/views/AccountPage.vue

```diff
@@ -1,68 +1,69 @@
+          </span>
+          <span class="validation col">
+            <button class="btn fas fa-check" v-on:click="updateName()"/>
+            <button class="btn fas fa-times" v-on:click="this.cancelEditing()"/>
+          </span>
+        </div>
+        <div v-else class="editableNameAccount">
+          <a v-on:click="this.displayTitleEditing()">
+            <h1>
+              {{ this.account ? this.account.name : "" }} :
+              {{ this.account ? getEurosAmount(this.account.amount) : "" }}€
+              <button class="btn fas fa-pen" />
+            </h1>
+          </a>
+        </div>
+        <div class="scrollable operationTable table-hover">
+          <div class="rectangleTop"/>
+          <OperationForm class="operationCreate" @update-operation-list="getAccountOperation" :accountId="this.accountId"/>
+          <template v-for="operation in this.operations" :key="operation">
+            <OperationForm v-if="operation.editing" @update-operation-list="getAccountOperation" :accountId="this.accountId" :operation="operation"/>
+            <div v-else class="operation storedOperation">
+              <div class="date col-2 offset-1">
+                <div>{{ $d(this.getDayAsDate(operation.day), "day") }}</div>
+              </div>
+              <div class="col-8"></div>
+              <div class="category col-3 offset-1">
+                {{ this.getCategoryById(operation.categoryId)?.name ?? $t("UNKNOWN_CATEGORY") }}
+              </div>
+              <div class="amount col-3 offset-2">
+                {{ this.getEurosAmount(operation.amount) }} €
+              </div>
+              <div class="action col-1">
+                <button class="btn fas fa-pen" v-on:click="setAsEditing(operation)" :title="$t('EDIT')"/>
+                <button class="btn fas fa-trash" v-on:click="deleteOperation(operation)" :title="$t('DELETE')"/>
+              </div>
+              <div class="memo col-3 offset-1">{{ operation.memo }}</div>
             </div>
-            <div class="action col-1 offset-1">
-              <button class="btn fas fa-pen" v-on:click="setAsEditing(operation)" :title="$t('EDIT')"/>
-              <button class="btn fas fa-trash" v-on:click="deleteOperation(operation)" :title="$t('DELETE')"/>
-            </div>
-            <div class="memo col-3 offset-1">{{ operation.memo }}</div>
-          </div>
-        </template>
-        <ul id="actionsList" class="list-group list-group-horizontal d-flex justify-content-center">
-            <li><button v-on:click="goHomePage" class="btn fas fa-home" :title="$t('HOME')"/></li>
-            <li><button class="btn fas fa-chart-line disabled" :title="$t('GRAPH_AND_REPORT')"/></li>
-            <li><button class="btn fas fa-cog disabled" :title="$t('PREFERENCES')"/></li>
-            <li><button v-on:click="logout" class="btn fas fa-sign-out-alt" :title="$t('LOGOUT')"/></li>
-          </ul>
+          </template>
+          <div class="rectangleBottom"/>
```

Comment:
is this the placeholder ? if so could you add a comment for future reader

Reply:
Do I write a comment on template directly ?

Reply:
and yes, is a placeholder like rectangleTop

Reply:
yes you could just add a comment inside or rename rectangle as placeholder. Renaming could be best in fact